Added theme-color param for mobile devices

Reijnn opened the pull request.

rhazdon commented:
Hi @Reijnn,

thank you very much for your pull request. 

I like the idea, but I think that we need a default, too. Either an automatic change between light and dark colors for `theme-color`(like the body does) or you have to check whether the config variable is used and then set the `meta-tag`. 

In my opinion the automatic change is the better solution. It would be quite nice to see the chrome theme changing when switching from light to dark and vice versa. 

What do you think?

Reijnn commented:
Great idea, almost got this working.

As a side note i've added something to the README.md to generate favicons.
Also updated the viewport meta tag to gain a higher Lighthouse score:

```
[user-scalable="no"] is used in the <meta name="viewport"> element or the [maximum-scale] attribute is less than 5.
Disabling zooming is problematic for users with low vision who rely on screen magnification to properly see the contents of a web page.
```

rhazdon commented:
Looks great and works very well. Thank you so much! :)
